site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,72 @@
+weight: 150
+doctypes: ["install"]
+toc: true
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+Follow the steps in this document to pull the NGINX Plus Gateway Fabric image from the F5 Docker registry into your Kubernetes cluster using your JWT token.
+
+{{<important>}}An NGINX Plus subscription certificate and key will not work with the F5 Docker registry. For NGINX Gateway Fabric, you must have an Connectivity Stack for Kubernetes subscription.
+
+To list the available image tags using the Docker registry API, you will also need to download the NGINX Plus Gateway Fabric (per instance) certificate (`nginx-repo.crt`) and the key (`nginx-repo.key`) from [MyF5](https://my.f5.com).{{</important>}}
+
+## Before You Begin
+
+You will need the following information from [MyF5](https://my.f5.com) for these steps:
+
+- A JWT Access Token (per instance) for NGINX Gateway Fabric from an active NGINX Gateway Fabric subscription.
+- The certificate (`nginx-repo.crt`) and key (`nginx-repo.key`) for each NGINX Gateway Fabric instance, used to list the available image tags from the Docker registry API.
+
+## Prepare NGINX Gateway Fabric
+
+1. Log into the [MyF5 Portal](https://my.f5.com/), navigate to your subscription details, and download the relevant .cert, .key and .JWT files.
+
+## Using the JWT token in a Docker Config Secret
+
+1. Create a Kubernetes `docker-registry` secret type on the cluster, using the JWT token as the username and `none` for password (as the password is not used).  The name of the docker server is `private-registry.nginx.com`.
+
+    ```shell
+    kubectl create secret docker-registry nginx-plus-registry-secret --docker-server=private-registry.nginx.com --docker-username=<JWT Token> --docker-password=none [-n nginx-gateway]
+    ```
+
+    It is important that the `--docker-username=<JWT Token>` contains the contents of the token and is not pointing to the token itself. Ensure that when you copy the contents of the JWT token, there are no additional characters or extra whitespaces. This can invalidate the token and cause 401 errors when trying to authenticate to the registry.
+
+1. Confirm the details of the created secret by running:
+
+    ```shell
+    kubectl get secret nginx-plus-registry-secret --output=yaml
+    ```
+
+---
+
+## Helm Deployment
+
+If you are using Helm for deployment, please refer to [Installation with Helm]({{< relref "installation/installing-ngf/helm.md" >}})
+
+
+---
+
+## Pulling an Image for Local Use
+
+If you need to pull the image for local use to then push to a different container registry, use this command:
+
+```shell
+docker login private-registry.nginx.com --username=<output_of_jwt_token> --password=none
+```
+
+Replace the contents of `<output_of_jwt_token>` with the contents of the JWT token itself.
+Once you have successfully pulled the image, you can then tag it as needed.
```

Comment:
You included a guide for pulling the image, which contains different steps than here. Can we just include this as a link in the Alternative Installation options to the guide you wrote?